packages/ts/react-crud/src/locale.ts

```diff
@@ -47,10 +48,15 @@ export class LocaleFormatter {
       minute: 'numeric',
     });
 
-    this.#number = new Intl.NumberFormat(locale, {
+    this.#integer = new Intl.NumberFormat(locale, {
       maximumFractionDigits: 0,
     });
 
+    this.#decimal = new Intl.NumberFormat(locale, {
+      maximumFractionDigits: 2,
+      minimumFractionDigits: 2,
+    });
```

Comment:
Could this be customizable?

Reply:
It could, but this PR is really just meant to provide a reasonable default formatting.